app/admin/user_addresses.rb

```diff
@@ -0,0 +1,19 @@
+ActiveAdmin.register UserAddress do
+  belongs_to :user
+  permit_params :user_id, :fullname, :address_line1, :address_line2, :city,
+                :state, :zipcode, :country
+
+  form do |f|
+    f.inputs do
+      f.input :user_id, as: :hidden, value: params[:user_id]
```

Comment:
@varyonic , I think it shoud work without this input , since user_id is already part of url